### ISSUE TITLE
Just a question

### ISSUE THREAD

**Issue by ramihub** [michenriksen/aquatone#188]
Hello mate and thanks a lot for your effort by creating this tool.
Is there any way that aquatone can be still used for scanning for hostile subdomain takeovers? 
like old  aquatone-takeover option ?

Regards !

**Comment by michenriksen** [comment]
Yes, Aquatone still checks for subdomain takeover vulnerabilities, it happens in the [url_takeover_detector agent](https://github.com/michenriksen/aquatone/blob/master/agents/url_takeover_detector.go) which will check all responsive URLs for takeover situations across 20 different services. 

If it detects a takeover situation, it will add a red tag to the page in the HTML report with the text _Domain Takeover_ which links to documentation on how to register the domain on the service.
